File: AtCoder_Beginner_Contest/001/b.py
m = int(input())

if m < 100:
    print("{0:02d}".format(0))
elif 100 <= m <= 5000:
    print("{0:02d}".format(int(m/1000*10)))
elif 6000 <= m <= 30000:
    print("{0:02d}".format(int(m/1000)+50))
elif 35000 <= m <= 70000:
    res = int((m/1000-30)/5+80)
    print("{0:02d}".format(res))
elif 70000 < m:
    print("{0:02d}".format(89))

File: AtCoder_Beginner_Contest/001/b.rb
m = gets.to_i
res = 0
if m < 100
  res=0
elsif m.between?(100,5000)
  res=m/100
elsif m.between?(6000,30000)
  res=m/1000+50
elsif m.between?(35000,70000)
  res=(m/1000-30)/5+80
elsif 70000 < m
  res=89
end
puts sprintf("%02d",res)
